### PR TITLE
Docs: Add validators to dynamic model

### PR DESCRIPTION
**Pull request by bsitruk**
## Checklist

* [ ] Unit tests for the changes exist
* [ ] Tests pass on CI and coverage remains at 100%
* [ ] Documentation reflects the changes where applicable
* [ ] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)

### CODE REVIEW

File: docs/examples/models_dynamic_validators.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+validators = {
+    'username_validator':
+    validator('username')(username_alphanumeric)
+}
+
+WithValidatorModel = create_model(
+    'WithValidatorModel',
+    username=(str, ...),
+    __validators__=validators
+)
```
Comment (samuelcolvin) [inline docs/examples/models_dynamic_validators.py:18]: Looks good, please could you add an example of this being used, e.g. a valid case and an error.

**Review by samuelcolvin — COMMENTED** [review]
otherwise looks great.

File: docs/examples/models_dynamic_validators.py (hunk 1 above)
Comment (bsitruk) [reply docs/examples/models_dynamic_validators.py:18]: 👍 Thanks for the feedback.

**Comment by samuelcolvin** [comment]
thanks so much.

**Pull request merged**